<launch>
	<node pkg="tms_ss_ibs" type="ibs.py" name="ibs_cbnt" output="screen">
		<param name="PORT_TR"  value="/dev/ttyUSB0" />
		<param name="PORT_LC0" value="/dev/ttyACM0" />
		<param name="frame_id" value="/cabinet_link" />
		<param name="idSensor" value="3002" />
		<param name="idPlace"  value="6019" />
		<param name="offset_x"		   value="0.0" />
		<param name="offset_y"		   value="0.0" />
		<param name="offset_z"		   value="0.8" />
		<rosparam param="loadcell_points/x">[0.04, 0.425, 0.04, 0.425]</rosparam>
		<rosparam param="loadcell_points/y">[0.047, 0.047, 0.27, 0.27]</rosparam>
	</node>
</launch>
